docs/pages/guides/adopting-prebuild.md

```diff
@@ -0,0 +1,128 @@
+
+Run the following command to regenerate the `/ios` and `/android` directories based on the `app.json` configuration.
+
+<Terminal cmd={[
+'$ npx expo prebuild --clean',
+]} cmdCopy="npx expo prebuild --clean" />
+
+You can test that everything worked by building the projects locally:
+
+<Terminal cmd={[
+'# Build your native Android project',
+'$ npx expo run:android',
+'',
+'# Build your native iOS project',
+'$ npx expo run:ios'
+]} />
+
+## Extra changes
+
+The following changes are optional but recommended.
+
+**.gitignore**
+
+You can add `.expo/` to your `.gitignore` to prevent generated values from Expo CLI from being committed. You can also add `ios/` and `android/` to the `.gitignore` if you want to ensure they aren't committed between prebuilds.
+
+**app.json**
+
+Remove all fields that are outside of the top-level `expo` object as these will not be used in `npx expo prebuild`.
+
+```diff
+{
+-  "name": "myapp",
+-  "displayName": "myapp"
++  "expo": {
++    "name": "myapp"
++  }
+}
+```
+
+**metro.config.js**
+
+- See [Customizing Metro](/guides/customizing-metro.md)
+
+**package.json**
+
+You may want to change the scripts to use the Expo CLI run commands:
+
+```diff
+  "scripts": {
+    "start": "expo start --dev-client",
+-    "android": "react-native run-android",
+-    "ios": "react-native run-ios",
++    "android": "expo run:android",
++    "ios": "expo run:ios",
+  },
+```
+
+These have better logging, auto code signing, better simulator handling, and they ensure you run `npx expo start` to host files.
+
+## Advanced Migration
+
+If you're project already has a number of custom native modifications then you'll need to configure your Expo config (`app.json`) to reflect those native changes. There is currently no automated system for doing this. So instead we'll provide some pointers:
+
+- Check to see if your changes overlap with the built-in [Expo config fields](/versions/latest/config/app/). For example, if you have an app icon, be sure to define it as `expo.icon` in the `app.json` then re-run `npx expo prebuild`.
+- Look up if any of the packages you're using require an [Expo Config Plugin][config-plugins]. If a package in your project required additional changes inside of the `ios/` or `android/` folders, then you will probably need a Config Plugin. Some plugins can be automatically added by running `npx expo install` with all of the packages in your `package.json` `dependencies`. If a package requires a plugin but doesn't supply one, then you can try checking the community plugins at [`expo/config-plugins`](https://github.com/expo/config-plugins) to see if one already exists.
+- You can use the [VS code Expo extension][vs-code-expo] to "introspect" your changes and debug if prebuild is generating the native code you expect. Just press `cmd+shift+P`, type "Expo: Preview Modifier", and select the native file you wish to introspect.
```

Comment:
```suggestion
- You can use the [VS Code Expo extension][vs-code-expo] to introspect your changes and debug if prebuild is generating the native code you expect. Just press `cmd+shift+P`, type "Expo: Preview Modifier", and select the native file you wish to introspect.
```

Let's not use quotes to highlight a text or a word. If you think it's important to highlight something, let's bold text.